setup.cfg

```diff
@@ -24,13 +24,13 @@ classifiers =
 [options]
 packages = find:
 setup_requires = setuptools_scm
-python_requires = >=3.8
+python_requires = >=3.11
```

Comment:
let's keep this the same as lstchain `python_requires = >=3.9` which means also we can install 3.11 (and I think it'll do by default)